### DISCUSSION TITLE
How can i know is there a merged cell?

### DISCUSSION THREAD

**Discussion by kuangyanit** [exceljs/exceljs#1685] [Q&A] [answered]

**Comment by andorep** [comment, marked as answer]
I have the same issue. 
So far I found that if a cell has the property `master`, It's a merged cell.

```
ws.mergeCells('A3:A4');  // mergeCells
console.log('A2-Model', ws.getCell('A2').model); // A2-Model { address: 'A2', type: 2, value: 1, style: {} }
console.log('A4-Model', ws.getCell('A4').model); // A4-Model { address: 'A4', type: 1, master: 'A3', style: {} }
```

source: #1025

**Reply by Siemienik** [reply]
That right, if the cell is merged, then has a master cell. 
The master cell is always top left corner.

Usage example: 

https://github.com/Siemienik/XToolset/blob/f6f0a7de85d719fb7937a19d61ca9a09c96dc6d8/packages/xlsx-renderer/src/cell/BaseCell.ts#L16-L28

Additionally, the cell has getter `isMerged`

so both are correct:

```js
if(ws.getCell(`A1`).isMerged)

if(ws.getCell(`A1`).master)
```
